### ISSUE TITLE
Flickering in Full Screen on macOS Ventura on M2

### ISSUE THREAD

**Issue by mfrancisbeehive** [a85/POSTMan-Chrome-Extension#11555]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

The entire app will sometimes flicker when I've been using it in full screen for a while on macOS Ventura. This issue never occurred before I upgraded to Ventura. I have a MacBook Pro with the M2 chip. I know this also occurs on other apps as well, but some developers (such as JetBrains) have fixed this issue in their apps. It does not occur when not in full screen.

### Steps To Reproduce

1. Use Postman in macOS Ventura in full screen for a while
2. Wait for app to start flickering while actively using it or moving the mouse around over it

### Operating System

macOS Ventura

### Postman Version

10.6.7

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by cemck** [comment]
Same issue on my end (with M1 macOS Ventura 13.1 and Postman v10.8.1)

**Comment by mfrancisbeehive** [comment]
Since I've submitted the issue I have reinstalled macOS Ventura, and still encounter the same issue.

**Comment by jviotti** [comment]
@mfrancisbeehive Interesting. What other apps apart from JetBrains have you seen experiencing this issue? I can't reproduce (on an M1 running Ventura) so if other apps are aware, that might give us a lead to investigate further.

@dsanders11 are you aware of this problem from the Electron point of view?

**Comment by mfrancisbeehive** [comment]
Besides JetBrains IDEs, the only other app I observe to have this issue is Google Chrome. 

EDIT: I also have observed the issue when using the Spotify desktop app @jviotti

**Comment by jviotti** [comment]
Right. The issue might be coming from Chromium then, which both Google Chrome and Postman uses (through Electron). Assigning to @dsanders11 to investigate further.

**Comment by cemck** [comment]
The issue seems to be appearing when I'm using a second monitor connected to my MacBook with the mouse cursor on the second screen. The flickering disappears when moving the cursor to the Postman screen. Maybe this will help to reproduce. No issues with Chrome though

Edit: Same issue with VSCode

**Comment by roitalpaz** [comment]
![image](https://user-images.githubusercontent.com/77916282/213120146-9270c3fa-9cff-4298-87b2-be4b56e2f82a.png)
this happens to me as well (i think its the same issue)

when using the second screen it happens, but when switching to the native mac screen all is working well for a while and then breaks again. to fix i move back to the second screen and back to the native screen.

also not errors with chrome

attaching a screen shot.

on Mac M1, 13.1 ventura

**Comment by mfrancisbeehive** [comment]
I have been unable to reproduce it as described. For me it usually continues to flicker when moving the mouse over the full screen window

**Comment by MC-sanderv** [comment]
It looks like a Ventura + GPU/Chromium issue rather than something Postman-specific and quite a few updates have been made to MacOS and Electron since then.

I’m not able to reproduce this on newer Postman versions or newer macOS releases, and there haven’t been any recent updates on this thread. I’m going to close this as not planned.

If anyone still sees similar flickering on a current Postman build and macOS version, please re-open the issue with your macOS version, Postman version, GPU details, and whether hardware acceleration or full-screen mode changes the behaviour.
